src/core/memory.cpp

```diff
@@ -587,7 +587,11 @@ void Memory::UnmapRegion(Common::PageTable& page_table, VAddr base, u64 size) {
 bool Memory::IsValidVirtualAddress(const VAddr vaddr) const {
     const Kernel::KProcess& process = *system.CurrentProcess();
     const auto& page_table = process.PageTable().PageTableImpl();
-    const auto [pointer, type] = page_table.pointers[vaddr >> PAGE_BITS].PointerType();
+    size_t page = vaddr >> PAGE_BITS;
```

Comment:
```suggestion
    const size_t page = vaddr >> PAGE_BITS;
```

To be consistent with the surrounding code

Reply:
Will change in a bit.